pkg/cover/backend/elf.go

```diff
@@ -171,22 +210,26 @@ func readSymbols(file *elf.File) ([]*Symbol, uint64, uint64, map[uint64]bool, er
 		if symb.Value < text.Addr || symb.Value+symb.Size > text.Addr+text.Size {
 			continue
 		}
+		start := symb.Value
+		if module.Name != "" {
```

Comment:
For the kernel Addr == 0, so this if is unnecessary as well:
```
start := symb.Value + module.Addr
```

Reply:
Actually for kernel module.Addr is _text address on target which is not 0.
So like for qemu amd64, module address is larger than kernel _text, but for arm64, module address is smaller than kernel _text.
So we need _text address in order to group PCs returned.